content/app-man-syslog-ng/syslog-ng-ctl.1.md

```diff
@@ -43,30 +43,24 @@ The `syslog-ng-ctl` application is a utility that can be used to:
 
 ## Enabling troubleshooting messages
 
-`command [options]`
+`syslog-ng-ctl log-level <level>`
 
-Use the `syslog-ng-ctl <command> --set=on` command to display verbose, trace, or debug messages. If you are trying to solve configuration problems, the verbose (and occasionally trace) messages are usually sufficient. Debug messages are needed mostly for finding software errors. After solving the problem, do not forget to turn these messages off using the `syslog-ng-ctl <command> --set=off` command. Note that enabling debug messages does not enable verbose and trace messages.
+Available in {{% param "product.abbrev" %}} 4.0 and later.
 
-Use `syslog-ng-ctl <command>` without any parameters to display whether the particular type of messages are enabled or not.
+Use the `syslog-ng-ctl log-level <level>` command to display verbose, trace, or debug messages. If you are trying to solve configuration problems, the verbose (and occasionally trace) messages are usually sufficient. Debug messages are needed mostly for finding software errors. After solving the problem, do not forget to return the log level to the default using the `syslog-ng-ctl log-level default` command.
```

Comment:
Verbose and debug is needed for config issues, trace is even more detailed than debug and that's what you might need to diagnose software errors.